src/services/codex-coin.js

```diff
@@ -0,0 +1,17 @@
+// import { contracts } from '@codex-protocol/ethereum-service'
```

Comment:
Comments still relevant?

Reply:
Oops, good catch. This was a side effect of a rushed commit.